server/catalog/table_options.h

```diff
@@ -144,6 +145,10 @@ struct Column {
 
   using Id = uint64_t;
 
+  static constexpr Id kFakeId = std::numeric_limits<Id>::max();
+  static constexpr std::string_view kFakeName = "@fake";
```

Comment:
`sdb_generated_pk`

Add test for table with same column name

Reply:
BTW good point - it might trigger name uniqueness assertion in analyzer